src/services/configServices/SettingsService.js

```diff
@@ -43,6 +50,37 @@ class SettingsService {
     }
   }
 
+  async getEncryptedPassword(sessionData) {
+    const { siteName } = sessionData
+    const siteInfo = await this.sitesService.getBySiteName(siteName)
+    if (siteInfo.isErr()) {
+      // Missing site indicating netlify site - return special result
+      return okAsync({
+        encryptedPassword: "",
+        iv: "",
+        isAmplifySite: false,
+      })
+    }
+    const { id, isPrivate } = siteInfo.value
+    if (!isPrivate)
+      return okAsync({
+        encryptedPassword: "",
+        iv: "",
+        isAmplifySite: true,
+      })
+
+    const deploymentInfo = await this.deploymentsService.getDeploymentInfoFromSiteId(
+      id
+    )
+    if (deploymentInfo.isErr()) return deploymentInfo
```

Comment:
Sanity check here, shouldnt we reuturn some sort of error here instead?

Reply:
This returns the wrapped error from `getDeploymentInfoFromSiteId` - we're not modifying it here